schema/CVE_Record_Format.json

```diff
@@ -98,6 +98,11 @@
             "minLength": 1,
             "maxLength": 1024
         },
+        "semver-2.0.0-version": {
```

Comment:
Since a PATCH is backwards compatible.
```suggestion
        "semver-2.0-version": {
```

Reply:
I may be incorrect, but I don't see this definition referenced anywhere. Is this intentional?